lib/elixir/lib/map_set.ex

```diff
@@ -63,6 +63,7 @@ defmodule MapSet do
 
   """
   @spec new(Enum.t) :: t
+  def new(enumerable)
```

Comment:
I am not 100% sure this is a strong convention, but I would prefer if we left a blank line between the bodyless clause you added here and the clauses below. :)